### PR TITLE
Thyroid Disease Prediction

### PR DESCRIPTION
**Pull request by Akash20x**
I successfully added Thyroid Disease Prediction model which will predict whether a person have normal condition(no thyroid disease) or hyperthyroid or hypothyroid. I applied different classification ,models and found decision tree works best for its classification. I also added files to implement in the webpage using flask so that user can use this application and predict thyroid by giving values to the attributes related to thyroid .

Closes: #58

### CODE REVIEW

**Review by yashvidas — APPROVED** [review]

File: models/ThyroidDisease/Thyroid_Disease_Prediction.ipynb [hunk 1]
```diff
@@ -0,0 +1,2436 @@
...
...WtZ+u+f3O/ssc+HhLiZJEANEBIZjbyTGrUx2proz6/TwIPXItV5nQI9doOESl36m3O+HnXv71N10jfYzUaxRIZFLsuroSH1Ty9YGu+x3VLPx6//BVmqj9bPMSP2cmp2Gj9nOTwe0JOvJyexhamiKRqp7CVKtTHQv7miVKCvr7HcWv1zz8es0j6FApNZbQZ6WlPO1p36k57r59+CNP3FS2rpcikdCgT5tSJ4qiA0KoZifHVB0rzuUQK8+p3798ZhNV5Xh+TmxACGb2L+zo2N+Dh3k0PjhyjfpvqjTa93bnsVrjnsGfsr3tdLa3nc6tjYfwX7mb21uOYGBpip76qbnMQJfaHZrxrIQL874O9bCqtzevQl+1OtU1i1eb1bLC0smWxEexL91Hm1fh59BTN6jesA46BnpIZFLqtGnIk/sRhZYP8DvKT73m8VOveQQfukojtSa5myOZL9EkV2tqNLg9wWq7hRy5RmN1fDce0oEQtVZTWyv6rpvGwWlrSQh9cbN37vNf2dBmKpvemM7+yd/z6PztlyaJQDUGe77Q9L0yjsEcOjXHw7cPvxXUl0gkNCpDm/26cOLHgyzyfp9F3u9z/fBl2g7yBMDBzZm0pNQC1yIqjOuHL9PAQzV7zcTClJr2NsQ+jC6zxr1+e5nSawpTek3hwqELdB2sSmw1cGtASlIK8TH5k6cjZ43E2NSYdQvX5fr9wqELNGvbDAAzCzNq2dfSzDYqK0/9QzC1l2Oi7l/s+nvw6HDu/uXR4Ws4qvuXer3didLuAyUS7Pq0IawCEkW7t+7Bt+ckfHtO4tyh83Qf3A2ARm4NSUlKJS4m/zpDY94fhbGpMasX5v4KX8zjGNzaq14rrOtUB10DvTIniQDWrN2qWZR69+5D+AxXzQ5q496CxGeJREXlHh/LZDKsrFSv9uno6NC7dzcCA1XrLO7efZBOndoBYGVlgbOzAyGhFbsgvaDqI2YU/ce5e/cuUqkUZ2dVcsff35969epV2PmCj/vj0NmFCae/Jistk/2zXnRIY/YvYbO3KuN++KMt9P56vOrzticDCFF/Mcdr0ShkejoM3ab6esTj60EcmreZlqO6Y25XkzemDuSNqQMB2OHzOamlWBgw6Lg/Tp1deff0crLTMtk968UXHv63/zPWe88F4MBHm+mn/txt8MkAgk5UfNZdmaPg1Mdb6bdtNlKZlNs
```
Comment (akshitadixit) [inline models/ThyroidDisease/Thyroid_Disease_Prediction.ipynb:2435]: @Akash20x this looks good, just make sure to add a short description of the model in the PR comment.

File: templates/thyroidDisease.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                                placeholder="FTI" required="required">
+                        </div>
+                    </div>
+                </div>
+                <input type="submit" class="btn btn-info btn-block" value="Predict Thyroid Disease">
+            </form>
+        </div>
+    </div>
+    <div class="col-md-2"></div>
+</div>
```
Comment (akshitadixit) [inline templates/thyroidDisease.html:152]: LGTM

**Review by akshitadixit — APPROVED** [review]

File: models/ThyroidDisease/Thyroid_Disease_Prediction.ipynb (hunk 1 above)
Comment (Akash20x) [reply models/ThyroidDisease/Thyroid_Disease_Prediction.ipynb:2435]: Done

**Comment by Akash20x** [comment]
@akshitadixit Please merge my pull request

**Comment by akshitadixit** [comment]
Thank you for the PR :)

**Pull request merged**